client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -131,26 +134,12 @@ class PacmanGame extends Component {
   }
 
   startGame = () => {
-    const { config, pacman, status } = this.state;
-    if (status === 1) {
-      let { score } = this.state;
-      clearInterval(this.animationHandler);
-      score -= 10;
-      this.setState({ status: 0, score });
-      return;
-    }
-    if (status === 2) {
-      this.setInitialGameState();
-      this.setState({
```

Comment:
Don't just remove it. We need it for the singleplayer

Reply:
So put it in a conditional. When I kept changes currentGameState() wasnt found.